components/formats-common/src/loci/common/NIOFileHandle.java

```diff
@@ -63,7 +63,7 @@
   //-- Static fields --
 
   /** Default NIO buffer size to facilitate buffered I/O. */
-  protected static int defaultBufferSize = 1048576;
+  protected static int defaultBufferSize = 16;
```

Comment:
That's a big difference. I wonder if we ought also to test some intermediate value?